src/vs/workbench/contrib/files/browser/fileCommands.ts

```diff
@@ -629,28 +629,31 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 			{
 				isOptional: true,
 				name: 'viewType',
```

Comment:
Can we set this name to be something meaningful now that it's not just view type